DataFormats/Luminosity/interface/PixelClusterCounts.h

```diff
@@ -0,0 +1,56 @@
+#ifndef DataFormats_Luminosity_PixelClusterCounts_h
+#define DataFormats_Luminosity_PixelClusterCounts_h
+/** \class reco::PixelClusterCounts
+ *  
+ * Reconstructed PixelClusterCounts object that will contain the moduleID, BX, and counts.
+ *
+ * \authors: Sam Higginbotham shiggib@cern.ch and Chris Palmer capalmer@cern.ch
+ * 
+ *
+ */
+#include <algorithm>
+#include <string>
+#include <sstream>
+#include <iostream>
+#include <vector>
+
+namespace reco {
+class PixelClusterCounts {
+static constexpr unsigned int nEmBX=3564;//Empty BX to fill with counts
```

Comment:
this can apparently be taken directly from DataFormats/Luminosity/interface/LumiConstants.h
```static const unsigned int numBX = 3564;       // number of BX per orbit```